packages/amplify-cli/src/commands/help.ts

```diff
@@ -1,5 +1,17 @@
+import { printer } from 'amplify-prompts';
 import { showAllHelp } from '../extensions/amplify-helpers/show-all-help';
 
+const TROUBLESHOOTING_PROJECT_URL = "https://docs.amplify.aws/project/troubleshooting/";
```

Comment:
I think 'amplify-cli-core' is a better place to put this

Reply:
moved this to cli-core